specs/interop/shared-lockbox.md

```diff
@@ -0,0 +1,138 @@
+# Shared Lockbox
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**
+
+- [Overview](#overview)
+- [Design](#design)
+  - [Interface and properties](#interface-and-properties)
+  - [Events](#events)
+- [Invariants](#invariants)
+  - [System level invariants](#system-level-invariants)
+  - [Contract level invariants](#contract-level-invariants)
+- [Architecture](#architecture)
+  - [Authorization Flow](#authorization-flow)
+  - [ETH Management](#eth-management)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## Overview
+
+With interoperable ETH, withdrawals will fail if the referenced `OptimismPortal` lacks sufficient ETH.
+This is due to having the possibility to move ETH liquidity across the different chains and it could happen
+that a chain ends up with more liquidity than its `OptimismPortal`.
+The `SharedLockbox` improves the Superchain's interoperable ETH withdrawal user experience and avoids this issue.
+To do so, it unifies ETH L1 liquidity in a single contract (`SharedLockbox`), enabling seamless withdrawals of ETH
+from any OP chain in the Superchain, regardless of where the ETH was initially deposited.
+
+## Design
+
+The `SharedLockbox` contract is designed to manage the unified ETH liquidity for the Superchain.
+It implements two main functions: `lockETH` for depositing ETH into the lockbox,
+and `unlockETH` for withdrawing ETH from the lockbox.
+
+These functions are called by the `OptimismPortal` contracts to manage the shared ETH liquidity
+when making deposits or finalizing withdrawals.
+
+Authorization of `OptimismPortal`s is managed by the `SuperchainConfig` contract when a chain
+is added to the dependency set.
+The `SharedLockbox` contract is proxied and managed by the L1 `ProxyAdmin`.
+
+### Interface and properties
+
+**`lockETH`**
+
+Deposits and locks ETH into the lockbox's liquidity pool.
+
+- The function MUST accept ETH.
+- Only authorized `OptimismPortal` addresses MUST be allowed to interact.
+- The function MUST NOT revert when called by an authorized `OptimismPortal`
+- The function MUST emit the `ETHLocked` event with the `portal` that called it and the `amount`.
+
+```solidity
+function lockETH() external payable;
+```
+
+**`unlockETH`**
```

Comment:
You should should specify where it actually withdraws the given amount of ETH to